### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by Mateo-Johnson** [Gold872/elastic-dashboard#24] [Q&A]
This should just work with the standard networktables commands right?

**Comment by Gold872** [comment]
I'm a bit confused? Do you mean like whether or not it will display data that is added to network tables with methods like `SmartDashboard.putNumber()`

It is able to display any data that is on network tables, and supports nearly every data type

**Reply by Mateo-Johnson** [reply]
yeah, sorry I should have been more clear. I was wondering if it worked with like sendable choosers and putnumber commands

**Reply by Gold872** [reply]
Yes it does, it supports most sendables and all `putNumber()`/`putBoolean()` methods
